api/v1beta2/ibmvpccluster_types.go

```diff
@@ -81,6 +81,7 @@ type VPCLoadBalancerSpec struct {
 	// +listType=map
 	// +listMapKey=port
 	// +optional
+	// ++kubebuilder:validation:UniqueItems=true
```

Comment:
Do mind sharing some reference about this tag usage.

Since its slice and for now we have only Port within it. what should be unique, Is port? in future we add another field name say PortName along with port then what should be unique is it port or port and portName combination?